### DISCUSSION TITLE
Option to setup reCAPTCHA to protect signup, login and feedback forms

### DISCUSSION THREAD

**Discussion by ivancarlosti** [pglombardo/PasswordPusher#2077] [Ideas]
/t

**Comment by pglombardo** [comment]
Hi @ivancarlosti - I agree/this is overdue.  Created #2078

**Comment by pglombardo** [comment]
Not an exact address to this FR but there is also https://github.com/markets/invisible_captcha

**Comment by pglombardo** [comment]
This is a start: https://github.com/pglombardo/PasswordPusher/pull/2173

Going with something simpler than recaptcha for the time being.

**Comment by pglombardo** [comment]
First release [v1.41.0](https://github.com/pglombardo/PasswordPusher/releases/tag/v1.41.0) is out with the captcha applied to the new user registration form only.  If no problems, I'll extend it to the other public forms.
